mentat/resources/templates/conversation_viewer.jinja

```diff
@@ -16,11 +16,31 @@
 <a class="download_parent"><span class="download_icon"></span></a>
                 {% for message in transcript["messages"] %}
                 <div class="message {{'user' if message["prior_messages"] is none else 'assistant clickable'}}">
-                    {{ message["message"] |e|replace('\n', '<br>')|safe }}
+                    {% if message["message"] is string %}
+                        {{ message["message"] |e|replace('\n', '<br>')|safe }}
+                    {% else %}
+                        {% for submessage in message["message"] %}
+                            {% if submessage["type"] == "text" %}
+                                {{ submessage["text"] |e|replace('\n', '<br>')|safe }}
+                            {% elif submessage["type"] == "image_url" %}
+                                <img src="{{ submessage['image_url']['url'] }}">
+                            {% endif %}
+                        {% endfor %}
+                    {% endif %}
                     {% if message["prior_messages"] is not none %}
                         {% for seen_message in message["prior_messages"] %}
                         <div class="message viewpoint-message {{seen_message['role']}} hidden">
-                            {{ seen_message['content'] |e|replace('\n', '<br>')|safe }}
+                            {% if seen_message["content"] is string %}
```

Comment:
There's some code duplication here. I tried to use mentat to make a macro to fix it and it failed. I tried myself and it seemed like the arguments to the macro were passed as plaintext. I don't think it's that important to fix but if anyone who has experience with jinja macros it'd be nice.

Reply:
I'll work on this